src/main.cpp

```diff
@@ -3010,6 +3010,26 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, CBlockIn
         }
     }
 
+    // Coinbase transaction must include an output sending 20% of
+    // the block reward to `FOUNDERS_REWARD_SCRIPT` until the first
+    // subsidy halving block, with exception to the genesis block.
+    if ((nHeight > 0) && (nHeight < consensusParams.nSubsidyHalvingInterval)) {
```

Comment:
These bounds exclude the genesis block (to be consistent with upstream) and the first halved block, so that's (currently) 210000 - 1 blocks * 10 ZEC / block == 2099990 ZEC total as Founder's Reward. We're not certain if that's acceptable based on business requirements, and we're proceeding with this for now.